src/main/java/com/hermesworld/ais/galapagos/events/TopicRemoveProducerEvent.java

```diff
@@ -0,0 +1,17 @@
+package com.hermesworld.ais.galapagos.events;
+
+import com.hermesworld.ais.galapagos.topics.TopicMetadata;
+import lombok.Getter;
+
+@Getter
+public class TopicRemoveProducerEvent extends TopicEvent {
+
+    private final String producerToBeDeletedId;
```

Comment:
Bad field and parameter name. The "toBeDeleted" is just not necessary here, this results from the context of the class. Just use "producerApplicationId".